module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -645,13 +645,14 @@ protected function getHoldingDetailsForItem($holding): array
      * Support method for getHolding() -- given a few key details, format an item
      * for inclusion in the return value.
      *
-     * @param string $bibId          Current bibliographic ID
-     * @param array  $holdingDetails Holding details produced by
-     * getHoldingDetailsForItem()
-     * @param object $item           FOLIO item record (decoded from JSON)
-     * @param int    $number         The current item number (position within
-     * current holdings record)
-     * @param string $dueDateValue   The due date to display to the user
+     * @param string $bibId            Current bibliographic ID
+     * @param array  $holdingDetails   Holding details produced by
+     *                                 getHoldingDetailsForItem()
+     * @param object $item             FOLIO item record (decoded from JSON)
```

Comment:
It looks to me like you're retrieving $item from a different API than the previous code used, which is necessitating some additional changes (like the way effective location ID is retrieved below). Should this comment reflect exactly where the item came from? Also, it seems like we'll need a changelog note about this, because this could be a breaking change if people have customized formatHoldingItem, and the values they rely on are no longer available or formatted differently. (I didn't dig in to see how many potential differences there are -- but it seems like a possible issue).